src/components/search/searchPageSearchForm/SearchPageSearchForm.tsx

```diff
@@ -0,0 +1,63 @@
+import {
+  Tag,
+  Button,
+  TextInput,
+  Select,
+  Checkbox,
+  IconSearch,
+} from "hds-react";
+
+import Text from "../../text/Text";
+import {
+  mockSportTypes,
+  mockArea,
+  mockPrice,
+  mockCategory,
+} from "./tmp/mockedData";
+import styles from "./searchPageSearchForm.module.scss";
+
+function SearchPageSearchForm() {
+  const handleSubmit = (e) => {
+    e.preventDefault();
+  };
+  return (
+    <div className={styles.searchArea}>
+      <Text variant="h1">Mitä etsit?</Text>
+      <form role="search" className={styles.form} onSubmit={handleSubmit}>
+        <TextInput
+          placeholder="Kirjoita hakusana, esim. uimahalli tai jooga"
+          id="text"
+          name="text"
+          className={styles.inputWithIcon}
+        >
+          <IconSearch aria-hidden="true" />
+        </TextInput>
+        <Select
+          label=""
+          id="place"
+          placeholder="Valitse paikka"
+          options={mockCategory}
+        />
+        <Select
+          label=""
+          id="sport"
+          placeholder="Valitse liikuntamuoto/laji"
+          options={mockSportTypes}
+        />
+        <Select label="" placeholder="Valitse alue" options={mockArea} />
+        <Select label="" placeholder="Valitse hinta" options={mockPrice} />
+
+        <Checkbox id="Box " label="Sopiva liikuntarajoitteisille" />
+        <Checkbox id="Box " label="Lähimmät paikat ensin" />
+        <Checkbox id="Box " label="Näytä vain ilmaiset" />
+        <Checkbox id="Box " label="Avoinna nyt" />
```

Comment:
IDs should be unique within the page. I.e. you should be able to find only a single element with a given ID.

Consider whether these checkboxes should be grouped with `SelectionGroup`.